Content.Server/Stack/Events/StackSplitAttemptEvent.cs

```diff
@@ -0,0 +1,33 @@
+namespace Content.Server.Stack.Events;
+
+public sealed class StackSplitAttemptEvent : CancellableEntityEventArgs
+{
+    /// <summary>
+    /// The item being Split by the user
+    /// </summary>
+    public EntityUid Used { get; }
+
+    /// <summary>
+    /// The User splitting the item
+    /// </summary>
+    public EntityUid User { get; }
+    public int Amount { get; }
+    public StackComponent? Stack { get; }
+    public TransformComponent? UserTransform { get; }
```

Comment:
do not store components on events like this

Reply:
So am I supposed to try to get those event in the system?

Reply:
Removed